### DISCUSSION TITLE
关于如何自定义双拼方案

### DISCUSSION THREAD

**Discussion by ghost** [fcitx5-android/fcitx5-android#388] [Q&A | 问答] [answered]
@rocka 请问该如何使用自定义双拼功能。

我在配置中看到了可以自定义双拼方案，但是在帮助文档上面没有找到相关说明。
我参考了下 https://github.com/fcitx/fcitx5-chinese-addons/issues/44 的说明，在 `\data\org.fcitx.fcitx5.android\files\data\pinyin` 目录下增加了 `sp.dat` 自定义双拼文件。文件内容如下：
```shell
[方案]
方案名称=自定义

[零声母标识]
=

[声母]
# 双拼编码就是它本身的声母不必列出
ch=u
ch=;
sh=i
zh=o
zh=v

[韵母]
# 双拼编码就是它本身的韵母不必列出
iu=q
ian=w
uan=r
in=t
ua=a
ua=m
ie=y
ou=o
uo=p
uo=b
a=a
ia=l
ong=l
ei=s
eng=d
ing=f
ang=g
iang=g
ai=h
ao=j
iao=j
en=k
iong=l
ue=;
ve=;
uang=z
uai=x
un=c
ui=;
v=v
an=n

[音节]
ju=jv
e=ee
e=e;
er=er
ei=ei
eng=eg
en=en
ou=ou
o=oh
o=o;
ai=ai

......

```
但是并没有生效。

**Comment by rocka** [comment, marked as answer]
桌面上 `~/.config/fcitx5/pinyin/sp.dat` 对应到 Android 上的位置是 `data/org.fcitx.fcitx5.android/files/config/pinyin/sp.dat` ，是 config ，不是 data

**Reply by ghost** [reply]
多谢，放在`data/org.fcitx.fcitx5.android/files/config/pinyin`目录生效了。
